src/main/java/io/temporal/internal/testservice/TestServiceRetryState.java

```diff
@@ -104,10 +105,10 @@ BackoffInterval getBackoffIntervalInSeconds(Optional<String> errorType, long cur
       // MaximumAttempts is the total attempts, including initial (non-retry) attempt.
       return new BackoffInterval(RetryState.RETRY_STATE_MAXIMUM_ATTEMPTS_REACHED);
     }
-    long initInterval = TimeUnit.SECONDS.toMillis(retryPolicy.getInitialIntervalInSeconds());
+    long initInterval = Durations.toMillis(retryPolicy.getInitialInterval());
```

Comment:
change these intervals/expirations to java.time.Durations?

Reply:
Only if it improves readability of this code.